omod/src/main/webapp/fragments/patientdashboard/encountertemplate/defaultEncounterTemplate.gsp

```diff
@@ -43,11 +43,11 @@
 
 	<span>
         {{ if ( config.editable && encounter.canEdit) { }}
-            <i class="editEncounter delete-item icon-file-alt" data-mode="view" data-patient-id="{{- patient.id }}" data-encounter-id="{{- encounter.encounterId }}" {{ if (config.viewUrl) { }} data-view-url="{{- config.viewUrl }}" {{ } }} title="${ ui.message("coreapps.view") }"></i>
-            <i class="editEncounter delete-item icon-pencil" data-patient-id="{{- patient.id }}" data-encounter-id="{{- encounter.encounterId }}" {{ if (config.editUrl) { }} data-edit-url="{{- config.editUrl }}" {{ } }} title="${ ui.message("coreapps.edit") }"></i>
+            <i class="viewEncounter editEncounter edit-action icon-file-alt" data-mode="view" data-patient-id="{{- patient.id }}" data-encounter-id="{{- encounter.encounterId }}" {{ if (config.viewUrl) { }} data-view-url="{{- config.viewUrl }}" {{ } }} title="${ ui.message("coreapps.view") }"></i>
```

Comment:
@dkayiwa Because you are viewing the encounter not editing it - I later noticed that the class names are built from UI commons which requires additional changes so I just added the viewEncounter class.